2021-08-19 14:11:18,417 INFO    MainThread:15915 [internal.py:wandb_internal():91] W&B internal server running at pid: 15915, started at: 2021-08-19 14:11:16.152382
2021-08-19 14:11:18,420 INFO    WriterThread:15915 [datastore.py:open_for_write():77] open: /tmldata1/saberika/advex-uar/advex_uar/wandb/offline-run-20210819_141036-1qtzjw4b/run-1qtzjw4b.wandb
2021-08-19 14:11:18,420 DEBUG   HandlerThread:15915 [handler.py:handle_request():94] handle_request: run_start
2021-08-19 14:11:20,987 DEBUG   HandlerThread:15915 [meta.py:__init__():34] meta init
2021-08-19 14:11:20,987 DEBUG   HandlerThread:15915 [meta.py:__init__():48] meta init done
2021-08-19 14:11:20,987 DEBUG   HandlerThread:15915 [meta.py:probe():190] probe
2021-08-19 14:11:21,243 DEBUG   HandlerThread:15915 [meta.py:_setup_git():180] setup git
2021-08-19 14:11:39,727 DEBUG   HandlerThread:15915 [meta.py:_setup_git():187] setup git done
2021-08-19 14:11:40,401 DEBUG   HandlerThread:15915 [meta.py:_save_pip():52] save pip
2021-08-19 14:11:44,085 DEBUG   HandlerThread:15915 [meta.py:_save_pip():66] save pip done
2021-08-19 14:11:44,086 DEBUG   HandlerThread:15915 [meta.py:probe():231] probe done
2021-08-19 14:16:41,899 WARNING MainThread:15915 [internal.py:wandb_internal():146] Internal process interrupt: 1
2021-08-19 14:16:43,952 WARNING MainThread:15915 [internal.py:wandb_internal():146] Internal process interrupt: 2
2021-08-19 14:16:43,953 ERROR   MainThread:15915 [internal.py:wandb_internal():149] Internal process interrupted.
2021-08-19 14:16:44,397 INFO    WriterThread:15915 [datastore.py:close():258] close: /tmldata1/saberika/advex-uar/advex_uar/wandb/offline-run-20210819_141036-1qtzjw4b/run-1qtzjw4b.wandb
2021-08-19 14:16:44,399 INFO    HandlerThread:15915 [handler.py:finish():333] shutting down handler
2021-08-19 14:16:44,727 INFO    SenderThread:15915 [sender.py:finish():766] shutting down sender
2021-08-19 14:16:45,406 INFO    MainThread:15915 [internal.py:handle_exit():78] Internal process exited
